molgenis-data-rest/src/main/java/org/molgenis/data/rest/v2/RestControllerV2.java

```diff
@@ -113,6 +116,24 @@ public RestControllerV2(DataService dataService, MolgenisPermissionService permi
 		this.restService = requireNonNull(restService);
 	}
 
+	@Autowired
```

Comment:
I need authorization to access, I don't think the version endpoint should be protected